include/bolgenos-ng/pic_common.h

```diff
@@ -15,7 +13,7 @@
 struct pic_device {
 	void (*setup)(); /*!< Function that initilizes PIC device. */
 
-	void (*end_of_interrupt)(irq_t irq); /*!< Function that send
+	void (*end_of_interrupt)(irq::irq_t irq); /*!< Function that send
```

Comment:
:exclamation: Namespaces are not acceptable in code on C. Change programming language for this header explicitly.